lib/storage/client.js

```diff
@@ -1044,6 +1044,30 @@ class Client extends CryptoConsumer {
     const params = { id_string: noteName }
     return shared.issueNoteChallenge(this, params, meta)
   }
+
+  /**
+   * createGroup creates a group and sends it to TozStore.
+   *
+   * @param {string} name  the name of the group
+   * @param {array} capabilities an array of strings of the capabilities the group will have
+   *
+   * @returns {Group} A response from TozStore; the group that has been written.
+   */
+  async createGroup(name, capabilities = []) {
+    /* eslint-enable */
+    const encryptionKeys = new KeyPair(
+      this.config.publicKey,
+      this.config.privateKey
+    )
+    return shared.createGroup(
```

Comment:
I didn't think of this before, but I am a bit curious why this ended up in the shared functions? You may have been following patters, but in general shared is for methods that are re-used accross objects or different contexts. So for instance with notes, there is 'anonymous' notes which do not require a client, and 'regular' notes which are written by a client. The code goes in shared so the code to write a note is the same no matter if it is written from the anonymous or regular contexts. I don't think we'll ever be creating a group anonymously, or allow this to run in an object other than a client. In that case we can skip shared and just define the behavior here in the client.